test/test_utils.rb

```diff
@@ -166,4 +166,18 @@ class TestUtils < JekyllUnitTest
     end
   end
 
+  context "The \`Utils.append_permalink_ending\` method" do
+    should "handle built-in permalink styles" do
+      assert_equal "/:basename/", Utils.append_permalink_ending("/:basename", :pretty)
```

Comment:
huh?  This particular value isn't user configurable... it's the default template used for page/collection permalinks.  Look at the code it is replacing... it's already hardcoded.  Now we're just making the suffix match what the user has in their `permalink` setting in `_config.yaml`.  Users can always provide their own per-page `permalink` value in the front matter, or the per-collection `permalink` setting in `_config.yaml`